Libraries/WebSocket/RCTWebSocketModule.m

```diff
@@ -99,6 +100,27 @@ - (void)dealloc
   [_sockets[socketID] send:message];
 }
 
+RCT_EXPORT_METHOD(sendBlob:(NSDictionary *)blob socketID:(nonnull NSNumber *)socketID)
+{
+  RCTBlobManager *blobManager = [[self bridge] moduleForClass:[RCTBlobManager class]];
```

Comment:
@javache is there any advantage for having it as an optional module? any ideas/tips how else we can do it?